src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      class="bigSearch"
+      data-qa="big"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <button type="submit" class="bigSearch">
+        <img
+          class="bigSearch"
+          alt="search"
+          src="/images/Search.svg"
+        >
+      </button>
+      <input
+        type="search"
+        name="search"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="bigSearch"
```

Comment:
```suggestion
        class="bigSearch"
```
створи різні класи - search, bigSearch та smallSearch. В search пропиши загальні класи для всіх інпутів, а в big та small пропиши окремі, які стосуються конкретно великого чи малого інпута, і використовуй комбінацію `class="search bigSearch"`, наприклад.

Це потрібно для того, щоб не дублювати код для обох елементів, які мають якісь спільні властивості